xla/service/gpu/runtime3/command_buffer_cmd.cc

```diff
@@ -175,6 +175,27 @@ CommandBufferCmd::Slices MemcpyDeviceToDeviceCmd::slices() {
   return {dst_, src_};
 }
 
+//===----------------------------------------------------------------------===//
+// MemallocOnDeviceCmd
```

Comment:
`AllocateCmd` and also regular `Allocate` API name for command buffer. `OnDevice` part is implicit.

Reply:
Fixed